### PR TITLE
Fix numeric settings not being respected

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
This is the conclusion of one of the rabbit holes encountered when trying to update browser to not use any deprecated cypher. It went something like this:
- Try to understand the query that the `getNeighbours` method runs (which contains deprecated cypher)
- The count that the query returns used to show a warning, but there was a logic error 
- After fixing the error, notice that we have no code showing the warning anyway, fix that
- Warning still doesn't show up. Turns out the maxNeighbours setting was typed as number but a string at runtime. 
- Notice that this string/number issue is true for a few other settings as well
- Type up SettingsDuck and make the getters parse the strings to ints
- And then finally we're back at updating the query to not use the deprecated syntax! (in this case `RETURN size((a)--())`)

Anyway a preview is available at http://respect_numeric_settings.surge.sh

### CODE REVIEW

File: src/shared/modules/settings/settingsDuck.ts [hunk 1]
```diff
@@ -91,26 +92,56 @@ export const getAllowCrashReports = (state: GlobalState): boolean =>
...
+}
+
+export const initialState: SettingsState = {
+  maxHistory: '30',
   theme: AUTO_THEME,
   initCmd: ':play start',
   playImplicitInitCommands: true,
-  initialNodeDisplay: 300,
-  maxNeighbours: 100,
+  initialNodeDisplay: '300',
```
[the remark this answers is not in the document]
Comment (OskarDamkjaer) [reply src/shared/modules/settings/settingsDuck.ts]: updated

**Review by nglgzz — APPROVED** [review]
Looks good to me :sparkles:

**Comment by OskarDamkjaer** [comment]
As per your offline comment I'll double check it working in 3.5 and then merge 🚀

**Comment by OskarDamkjaer** [comment]
All good, thanks for the review

**Pull request merged**